hera_mc/mc_session.py

```diff
@@ -723,3 +724,178 @@ def get_paper_temps(self, starttime, stoptime=None):
 
         return self._time_filter(PaperTemperatures, 'time', starttime,
                                  stoptime=stoptime)
+
+    def add_ant_metric(self, obsid, ant, pol, metric, val):
+        """
+        Add a new antenna metric to the M&C database.
+
+        Parameters:
+        ------------
+        obsid: long integer
+            observation identification number
+        ant: integer
+            antenna number
```

Comment:
I think we need to think a little about what we actually want here, it turns out that 'antenna number' isn't a very precise term in M&C :). I do think we want something that is tied to the physical antenna, not the correlator input number, but I'm not quite sure whether it should be a 'station' or something else. @david-deboer?

I also wonder whether we want it to be a ForeignKey into one of the cm tables.